lib/topological_inventory/openshift/operations/worker.rb

```diff
@@ -36,37 +35,40 @@ def stop
 
         private
 
-        attr_accessor :messaging_client_opts, :client, :api_client, :sleep_poll, :poll_timeout
+        attr_accessor :messaging_client_opts, :client, :sleep_poll, :poll_timeout
 
         def process_message(client, msg)
           logger.info("Processing #{msg.message} with msg: #{msg.payload}")
           # TODO: Move to separate module later when more message types are expected aside from just ordering
-          order_service(client, msg)
+          order_service(msg.payload)
+          client.ack(msg.ack_ref)
         rescue StandardError => e
           logger.error(e.message)
           logger.error(e.backtrace.join("\n"))
           nil
         end
 
-        def order_service(client, msg)
-          task_id, service_plan_id, order_params = msg.payload.values_at("task_id", "service_plan_id", "order_params")
+        def order_service(payload)
+          task_id, service_plan_id, order_params = payload.values_at("task_id", "service_plan_id", "order_params")
 
           service_plan     = api_client.show_service_plan(service_plan_id)
           service_offering = api_client.show_service_offering(service_plan.service_offering_id)
+          source_id        = service_plan.source_id
 
-          catalog_client = Core::ServiceCatalogClient.new(service_plan.source_id)
+          catalog_client = Core::ServiceCatalogClient.new(source_id)
 
-          logger.info("Ordering #{service_offering.name} #{service_plan.name}...")
           service_instance = catalog_client.order_service_plan(
             service_plan.name, service_offering.name, order_params
           )
-          client.ack(msg.ack_ref)
-          logger.info("Ordering #{service_offering.name} #{service_plan.name}...Complete")
-
-          context = svc_instance_context_with_url(service_offering, service_plan, service_instance )
-          status  = provisioning_status(service_instance)
-
-          update_task(task_id, :state => "completed", :status => status, :context => context)
+          service_instance_name      = service_instance.metadata.name
+          service_instance_namespace = service_instance.metadata.namespace
```

Comment:
Also super minor, but these intermediate variables are not needed because they can just be part of the method call below without loss of readability.

Reply:
Done